gaphor/UML/actions/activitynodes.py

```diff
@@ -179,22 +177,9 @@ def __init__(self, diagram, id=None):
         self.watch("subject[NamedElement].name")
         self.watch("subject.appliedStereotype.classifier.name")
 
-    def save(self, save_func):
-        if self._combined:
-            save_func("combined", self._combined)
-        super().save(save_func)
-
-    def load(self, name, value):
-        if name == "combined":
-            self._combined = value
-        else:
-            super().load(name, value)
-
-    @observed
-    def _set_combined(self, value):
-        self._combined = value
-
-    combined = reversible_property(lambda s: s._combined, _set_combined)
+    combined: relation_one[UML.ControlNode] = association(
```

Comment:
I like how adding these as associations reduced the number of methods needed :+1: